spinn_front_end_common/interface/interface_functions/pre_allocate_resources_for_live_packet_gatherers.py

```diff
@@ -43,62 +41,19 @@ def __call__(
         :rtype: ~pacman.model.resources.PreAllocatedResourceContainer
         """
 
-        progress = ProgressBar(
-            len(machine.ethernet_connected_chips),
-            "Preallocating resources for Live Recording")
+        progress_bar = ProgressBar(
+            1, "Preallocating resources for Live Recording")
```

Comment:
Should this iterate over the number of LPG parameters?

Reply:
in theory yes over the number of LPG parameters? would work.

But considering that each pass is very fast and if there are more than 17  LPG parameters there is a much bigger problem just deleted the bar.